app/viewmodels/govuk/ErrorSummaryFluency.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.viewmodels.govuk
+
+import play.api.data.Form
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.errorsummary.{ErrorLink, ErrorSummary}
+
+object errorsummary extends ErrorSummaryFluency
+
+trait ErrorSummaryFluency {
+
+  object ErrorSummaryViewModel {
```

Comment:
this is not best practice to hide singleton within trait, I would think singleton should be outside of the trait??

Reply:
I think if you would analyse this file, its seems like custom package object implementation, there is no need for these complexities , at least in our code base; in case you keen to use package object that fine